modules/openxr/register_types.cpp

```diff
@@ -139,6 +139,7 @@ void initialize_openxr_module(ModuleInitializationLevel p_level) {
 		XRServer *xr_server = XRServer::get_singleton();
 		if (xr_server) {
 			openxr_interface.instantiate();
+			Engine::get_singleton()->add_singleton(Engine::Singleton("OpenXRInterface", openxr_interface.ptr(), "OpenXRInterface"));
```

Comment:
Ah, here it's added... Let's not do this. Registered singletons need to be guaranteed to exist, OpenXRInterface only exists when used.

If you need access to the interface from GDScript simply access `XRServer.primary_interface`

Makes the code more portable if we ever want the editor to work with other interfaces in the future as well.

Reply:
Fixed. This was done to address a bug caused by the creation of two `OpenXRInterface` instance. I used a different approach to resolve the bug without making `OpenXRInterface` a registered singleton.

Reply:
Ah easy :)